monkey_island/windows/openssl.cfg

```diff
@@ -0,0 +1,350 @@
+#
```

Comment:
Unsure why, but some of my commits appear under this PR. After you finish fixing all of the other comments, we should fix this (either by creating a new PR, back-merging from develop, etc...)